browser.py

```diff
@@ -20,27 +20,27 @@
 
 import gi
 gi.require_version("Gtk", "3.0")
-gi.require_version("WebKit", "3.0")
+gi.require_version("WebKit2", "4.0")
 
 from gi.repository import Gtk
 from gi.repository import GLib
-from gi.repository import WebKit
+from gi.repository import WebKit2 as WebKit
 from gi.repository import GObject
 
 
 class Browser(Gtk.ScrolledWindow):
 
     __gsignals__ = {
         "load-finished": (
-            GObject.SIGNAL_RUN_LAST, GObject.TYPE_NONE, [GObject.TYPE_STRING]
+            GObject.SIGNAL_RUN_LAST, None, (GObject.TYPE_STRING,)
```

Comment:
Can you explain why you changed GObject.TYPE_NONE to None, and why you changed the third argument from a list to a tuple?  I've looked at other activities, and GObject.TYPE_NONE is valid, and the third argument is usually a list inside a tuple.  In your debugging tests, does the signal get called?

Reply:
> Can you explain why you changed GObject.TYPE_NONE to None,

Correct me if I'm wrong but they both do the same thing, right?

>  and why you changed the third argument from a list to a tuple?

Didn't understand it wasn't meant to be.

>In your debugging tests, does the signal get called?

No.

Reply:
>  > Can you explain why you changed GObject.TYPE_NONE to None,

> Correct me if I'm wrong but they both do the same thing, right?

Yes, it looks like they have the same effect, but it isn't clear why, and as you changed it, and you asked for a review; my review is "why did you change it?"  Since, you've changed it back.

Reply:
I changed it because there's no need to have more characters when less characters does the job.